wordpress-mobile/WordPress-iOS
# Issues/7272 login ui tweaks

Refs #7272 

This PR includes updates to the login storyboard, adjusting the view hierarchy and constraints for presenting the login screens on an iPad.  Its also enforces portrait orientation on the prologue and epilogue screens.
- Screens on the iPad are limited to the same width as the iPhone 6s -- 383px.
- Constraints are adjusted so text fields are centered vertically in their parent view's container.  A top constraint is added to ensure the form does not get pushed above the top of the screen.  These changes remove the need for the `verticalCenterConstraint` used by the old sign-in vcs. 

Note: The epilogue remains full screen in this PR. Its stubbornly ignoring settings to follow readable content margins.  I'll sort that out in a follow up PR so as to not delay these changes. 

To test:
Test the login flow on both an iPad and iPhone.  Ensure that margins and placement are correct for both platforms. 

Needs review: @nheagy 

Examples:

![simulator screen shot jul 10 2017 1 57 29 pm](https://user-images.githubusercontent.com/1435271/28034921-523a0cdc-6578-11e7-87ab-fa545afe5e79.png)
![simulator screen shot jul 10 2017 1 57 38 pm](https://user-images.githubusercontent.com/1435271/28034933-557f0582-6578-11e7-8109-0d71a7048ac5.png)
![simulator screen shot jul 10 2017 2 01 03 pm](https://user-images.githubusercontent.com/1435271/28034934-5764d908-6578-11e7-86bd-26fa3058a6d8.png)
![simulator screen shot jul 10 2017 2 04 57 pm](https://user-images.githubusercontent.com/1435271/28035113-033a9a74-6579-11e7-8f93-907fa088dcf5.png)
![simulator screen shot jul 10 2017 2 09 53 pm](https://user-images.githubusercontent.com/1435271/28035264-afa09782-6579-11e7-81d7-3e9972113c64.png)
